conf.py

```diff
@@ -19,7 +19,7 @@
 project = u'Common Package Specification'
 copyright = u'2024, Matthew Woehlke'
 
-version_info = (0, 10, 0)
+version_info = (0, 11, 0)
```

Comment:
Orthogonal to this change, but @dcbaker have you used python-bumpversion before? It could help with automating this *and* avoiding some of the Meson-specific codegen in the config.h.

Reply:
Yes, I rather like python bump-version